dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/encryption/AuditCrypto.java

```diff
@@ -2085,6 +2085,10 @@ static final byte[] decrypt(byte[] mesg, byte[] key, String cipher, boolean useJ
             } catch (java.security.InvalidAlgorithmParameterException e) {
                 Tr.error(tc, "security.ltpa.noalgorithm", new Object[] { e });
                 com.ibm.ws.ffdc.FFDCFilter.processException(e, "com.ibm.ws.security.auditAuditCrypto", "2408");
+            } catch (NoSuchProviderException e) {
+                // TODO Auto-generated catch block
+                // Do you need FFDC here? Remember FFDC instrumentation and @FFDCIgnore
+                e.printStackTrace();
```

Comment:
Should this look like the catch above it instead of doing printStackTrace()?

Reply:
Fixed it.